streampark-common/src/main/scala/org/apache/streampark/common/conf/ConfigConst.scala

```diff
@@ -112,6 +112,8 @@ object ConfigConst {
 
   val KEY_FLINK_DEPLOYMENT_OPTION_PREFIX = "flink.deployment.option."
 
+  val KEY_JOBMANAGER_ARCHIVE_FS_DIR = "jobmanager.archive.fs.dir"
```

Comment:
This code can be deleted, we can use `JobManagerOptions.ARCHIVE_DIR.key()` directly.